src/Network/Network/help/New-AzFirewallPolicyNetworkRule.md

```diff
@@ -77,6 +78,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DestinationIpGroup
+The destination ipgroups of the rule
+
+```yaml
+Type: String[]
+Parameter Sets: (All)
+Aliases:
+
+Required: True
```

Comment:
this should be false, can you fix it manually

Reply:
Fixed